.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New issue
+about: 'Create a new issue'
+title: ''
+labels: ''
+assignees: ''
+
+---
+(Please remove the text below after reading it.)
+
+When filing an issue in this repository, please consider the following:
+
+- Is this:
+  - A feature request for a new form of integration etc?
+  - A report of a technical issue with the existing spec?
+  - A suggestion for improving the clarity of the existing spec (even if it's not "wrong" as such)?
```

Comment:
can you wrap things at 80 chars to be consistent with everything else in the repo?